crates/swc_ecma_transforms_react/src/jsx/mod.rs

```diff
@@ -360,13 +359,12 @@ impl JsxDirectives {
                             if let Some(src) = val {
                                 if is_valid_for_pragma(src) {
                                     // TODO: Optimize
```

Comment:
Maybe this todo was talking about the cloning? Perhaps we can remove this now?

Reply:
I think so